test/exports_test.cc

```diff
@@ -48,47 +48,60 @@ class TestContext : public ContextBase {
 };
 
 TEST_P(TestVM, Environment) {
+  // Initialize VM.
```

Comment:
Could you do this refactoring as part of a separate PR? It doesn't look necessary here and it makes reviewing this specific change much harder when it's intermixed with unrelated changes. Thanks!

Reply:
yeah right. Will revert the test refactoring change. Thanks for your patience.